spark/src/cgcloud/spark/spark_box.py

```diff
@@ -33,7 +34,7 @@
 
 hdfs_replication = 1
 
-hadoop_version = '2.6.0'
+hadoop_version = '2.6.2'
```

Comment:
Ah, sorry! Didn't mean to sneak such in. But, I would prefer to bump to Hadoop 2.6.2 as that's what we're running yonder here at Berkeley.